js/src/common/helpers/icon.tsx

```diff
@@ -1,11 +1,13 @@
+import { Attributes, Vnode } from 'mithril';
```

Comment:
Tbh, I would prefer sticking to `Mithril.Vnode` and `Mithril.Attributes` just for sake of consistency.

Reply:
Done